modules/abstract-eth/src/abstractEthLikeNewCoins.ts

```diff
@@ -33,20 +80,241 @@ interface HopPrebuild {
   gasPriceMax: number;
 }
 
+/**
+ * The extra parameters to send to platform build route for hop transactions
+ */
+interface HopParams {
+  hopParams: {
+    gasPriceMax: number;
+    userReqSig: string;
+    paymentId: string;
+  };
+  gasLimit: number;
+}
+
+export interface EIP1559 {
+  maxPriorityFeePerGas: number;
+  maxFeePerGas: number;
+}
+
+export interface ReplayProtectionOptions {
+  chain: string | number;
+  hardfork: string;
+}
+
 export interface TransactionPrebuild extends BaseTransactionPrebuild {
   hopTransaction?: HopPrebuild;
   buildParams: {
     recipients: Recipient[];
   };
   recipients: TransactionRecipient[];
-  nextContractSequenceId: string;
+  nextContractSequenceId: number;
```

Comment:
BREAKING CHANGE: Type of nextContractSequenceId is changed to number to avoid typescript error of incompatible function params for signTransaction method in AbstractEthLikeNewCoins and AbstractEthLikeCoin